src/palakSJun20/FirstLastIndexOfCharacters.java

```diff
@@ -0,0 +1,37 @@
+package palakSJun20;
+/* Write a program which print first and last index of every characters only if character has multiple occurrence.
+Input : technocredits
+Output :
+t -> first index = 0 Last index = 11
+e -> first index = 1 Last index = 8
+c -> first index = 2 Last index = 6 */
+
+public class FirstLastIndexOfCharacters {
+
+	//Method for Finding first and last index of every characters
+	void findFirstAndLastIndex(String input){
+		
+		for (int index = 0; index < input.length(); index++) {
+			char ch = input.charAt(index);
+			int count = 0;
+				if(index == input.indexOf(ch)) {
+					for(int i = 0; i < input.length(); i++) {
+						if (input.charAt(i) == ch) {
+							count++;
```

Comment:
You have increased the complexity in code, Using one for loop you can achieve the output.

Reply:
I m not sure it is correct way or not please check it took me more time to write new logic. :( 
didn't remove the previous method just commented method Calling and create new method. 
`void FirstAndLast(String input){
		for (int index = 0; index < input.length(); index++) {
			char ch = input.charAt(index);
			int count = 0;	
			if( index == input.indexOf(ch) && input.indexOf(ch) < input.lastIndexOf(ch)  )
				System.out.print(ch+ " -> First Index = "+ input.indexOf(ch) + " Last Index = "+ input.lastIndexOf(ch) +"\n");
		}
	}`

Reply:
Np, It's completely fine.